tests/sqlparser_common.rs

```diff
@@ -7596,12 +7598,12 @@ fn parse_uncache_table() {
     );
 }
 
-#[test]
-fn parse_deeply_nested_parens_hits_recursion_limits() {
-    let sql = "(".repeat(1000);
-    let res = parse_sql_statements(&sql);
-    assert_eq!(ParserError::RecursionLimitExceeded, res.unwrap_err());
-}
+// #[test]
```

Comment:
Ouch; that was entirely unintentional, I didn't mean to commit; on one of my builds it would panic, so I commented it out locally to make sure that everything else was passing. Will reinstate it!

Reply:
Fixed; sorry about that one ;)